capture_loop: false
capture_path: /src/tests/data_files/ptp_synced_x90.pcap
expected_sensor_ips: []
half_frequency_mode: false
include_blocked_points: true
include_invalid_points: false
include_noise_points: false
include_saturated_points: true
include_second_return_points: true
max_altitude: 90.0
max_azimuth: 90.0
max_distance: 1000.0
min_altitude: -90.0
min_azimuth: -90.0
min_distance: 0.0
output_by_handle: true
output_by_sn: true
using_cepton_coordinate_system: true
